.azure/azure-pipelines.periodic.yml

```diff
@@ -0,0 +1,62 @@
+#
+# Continuous Integration (CI)
+# This pipeline builds and runs MsQuic performance RPS long run tests.
+#
+
+trigger: none
+pr: none
+
+name: 0.$(Date:yyyy).$(Date:MM).$(DayOfMonth).$(Rev:rr).0
+stages:
+
+schedules:
+- cron: "0 0 * * Sun"
```

Comment:
This is the same day as the Release builds, right? Maybe we do this Saturday so as to not conflict?

Reply:
Nope. Release builds are Mon, so sunday evening. These will run saturday evening.